client/my-sites/checkout/composite-checkout/payment-methods/credit-card/credit-card-fields.js

```diff
@@ -77,24 +74,10 @@ export default function CreditCardFields() {
 		setCardDataError( input.elementType, null );
 	};
 
-	const contactCountryCode = useSelect(
-		( select ) => select( 'wpcom' )?.getContactInfo().countryCode?.value
-	);
-	const shouldShowContactFields =
-		contactCountryCode === 'BR' &&
-		Boolean(
-			cart?.allowed_payment_methods?.includes(
-				translateCheckoutPaymentMethodToWpcomPaymentMethod( 'ebanx' )
-			)
-		);
+	const shouldShowContactFields = shouldUseEbanx;
 	const { formStatus } = useFormStatus();
 	const isDisabled = formStatus !== FormStatus.READY;
 
-	// Cache the country code in our store for use by the processor function
-	useEffect( () => {
```

Comment:
Nice work. As we talked about in our pairing session, this comment was not accurate anyway.